resources/definitions/fdmprinter.def.json

```diff
@@ -5127,6 +5127,18 @@
                         }
                     }
                 },
+                "support_infill_density_multiplier_layer_0":
+                {
+                    "label": "Support Infill Density Multiplier First Layer",
```

Comment:
We're calling it 'Initial' layer everywhere else in the English names. Although this is also meant for the raft... but then it's not a single layer anymore.

Reply:
Yeah it's sometimes called "initial" and sometimes "first", so I'm never sure which one to use :smiley: but if you're sure Initial is the official word, I'll change it.

Reply:
In the _setting names_ at least, it's initial. In documentation etc. it's sometimes first I think.

Reply:
Oops, I though the "official" setting name for things related to first layers was "layer_0", because there are many occurences of this. I'll change it.